chainer/utils/conv_nd_kernel.py

```diff
@@ -28,9 +29,9 @@ def aux(exp, arg):
         return functools.reduce(aux, zip(xs, ys))
 
 
-def _map(fn, *lst):
+def map(fn, *lst):
```

Comment:
I think map is not appropriate because it hides builtin `map` function. So how about `map_`?